src/_guides/language/sound-problems.md

```diff
@@ -79,12 +79,7 @@ How you troubleshoot strong mode depends on whether you are running
 
 #### Command line analyzer
 
-If you are running `dartanalyzer --strong` and you don't see expected
-strong mode errors, be sure that you didn't disable strong mode in an
-analysis options file in the same directory where you run the analyzer.
-If you do, the analysis options file overrides your command line flag.
-
-For more information on how to set up an analysis options file, see
+For information on how to set up an analysis options file, see
```

Comment:
This doesn't really troubleshoot the problem, and it doesn't talk about command-line options at all. I'm not sure what this should say, though.

It kind of seems like it should say the opposite of what it said before: If you specify strong mode in your analysis options file, (1) make sure you specified it correctly and (2) make sure you didn't turn off strong mode using a command-line flag.

But it'd require a little testing to know whether that's correct.